### PR TITLE
NEPT-717: Advanced PIWIK rules feature - MASTER

### PR DESCRIPTION
**Pull request by netlooker**
This is follow up PR related to the https://github.com/ec-europa/platform-dev/pull/1259 PR.

### CODE REVIEW

File: profiles/common/modules/custom/nexteuropa_piwik/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+Next Europa PIWIK
+=================
+
+This feature provides the Piwik web statistics tracking system integration.
+
+Usage
+=====
+On the settings page you need to enter the Piwik website ID and define
```
Comment (Fefaine) [inline profiles/common/modules/custom/nexteuropa_piwik/README.md]: Please write here the location of the settings page

File: profiles/common/modules/custom/nexteuropa_piwik/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+On the configuration page click on the "Advanced PIWIK rules" tab which is located
+at the bottom of the page. 
+On the tab section tick the "Enable advanced PIWIK rules" checkbox and
+after that click on the "Save configuration" button.
+The notification with the following message should be displayed:
+"The PIWIK advanced rules are turned on."
+A new configuration tab "Advanced PIWIK rules" should become visible.
+
+#### Custom PIWIK rule types <a name="rule-types"></a>
```
Comment (Fefaine) [inline profiles/common/modules/custom/nexteuropa_piwik/README.md]: I don't see you mention that the cache should be cleaned for it to work

File: profiles/common/modules/custom/nexteuropa_piwik/README.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+If for a given page there will be more than one rule a log entry will be
+created with the information about overlapping rules IDs.
+
+The "Direct path" rules have a priority over the "Regexp path".
+It means that if two rules or more are applicable for the given page and
+one of them is the "Direct path" rule, that rule will be applied.
+Similar rule applies to the language selection. If there is more
+than a one rule, the rule with the defined language will be selected 
+instead of the one with an 'all' languages option.
+
```
Comment (Fefaine) [inline profiles/common/modules/custom/nexteuropa_piwik/README.md]: You don't mention here that if no rules apply the default section of "general settings" will apply

**Comment by netlooker** [comment]
I'm closing this PR.

**Pull request closed without merging**